### PR TITLE
Logging update secure forward master

### PR DESCRIPTION
**Pull request by mburke5678**
Replacement for https://github.com/openshift/openshift-docs/pull/17331

### CODE REVIEW

**Comment by mburke5678** [comment]
@richm @anpingli PTAL. I apologize for the confusion.  I *think* this is a better presentation of the material.

File: modules/cluster-logging-collector-external.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 = Configuring Fluentd to send logs to an external log aggregator
 
 You can configure Fluentd to send a copy of its logs to an external log
-aggregator, and not the default Elasticsearch, using the `forward`
+aggregator, and not the default Elasticsearch, using the *out-forward*
```
Comment (richm) [inline modules/cluster-logging-collector-external.adoc]: *out_forward* - underscore, not dash - https://docs.fluentd.org/output/forward "The out_forward Buffered Output plugin forwards events to other fluentd nodes"

File: modules/cluster-logging-collector-external.adoc [hunk 2]
```diff
@@ -19,12 +19,11 @@ The `forward` plug-in is supported by Fluentd only.
 The *in_forward* plug-in implements the server (receiving) side, and *out_forward* implements the client (sending) side.
 endif::openshift-enterprise[]
 
-To configure forwarding, edit the `secure-forward.conf` section in the Fluentd configmap in the `openshift-logging` project. On the external aggregator, configure the Fluentd `secure-forward.conf` file.
+To configure forwarding, edit the `secure-forward.conf` section in the Fluentd configmap in the `openshift-logging` project. On the external aggregator, configure the Fluentd `secure-forward.conf` file. For more information on using the *out-forward* plugin, see the link:https://docs.fluentd.org/output/forward[Fluentd documentation].
```
Comment (richm) [inline modules/cluster-logging-collector-external.adoc]: out_forward

**Comment by mburke5678** [comment]
@anpingli PTAL. Follow-up to https://github.com/openshift/openshift-docs/pull/17331 for more clarity and organization.

**Comment by richm** [comment]
lgtm

**Comment by mburke5678** [comment]
@anpingli PTAL.

**Review by anpingli — COMMENTED** [review]
@QiaolingTang  PTAL

**Comment by QiaolingTang** [comment]
LGTM

**Pull request merged**